### ISSUE TITLE
Allow polyline physics shape in spritebuilder

### ISSUE THREAD

**Issue by clickstan** [spritebuilder/SpriteBuilder#1583]
A simple addition, useful to add performance on top of polygon: allow an option for POLYLINE in the physics shape options. This could be grouped with the polygon, so the points are the same. Almost no code changes needed, just to keep all the polygon points without breaking them into smaller polygons. 

Also great for other uses, like being able to parse the polygon points for other in-game uses. (Right now, only the broken-up polygons can be parsed, making this difficult)

![screen shot 2015-04-22 at 11 32 51 am](https://cloud.githubusercontent.com/assets/8376113/7278458/9b15d71a-e8e3-11e4-816b-f1fca0bd6872.png)

**Comment by collidernyc** [comment]
I agree with this - Polyline option would be super useful. Thanks for adding this @clickstan

**Comment by clickstan** [comment]
I agree, and would be a quick addition.

**Comment by Lazzl0** [comment]
I agree as well. This would be great and would save a lot of time and effort put into level building.

**Comment by pinestein** [comment]
I agree with @clickstan and the rest of you. This would be very useful in a number of cases. It would be awesome to hear if this is on the roadmap, like stan said - the data is already there :)
